api/db/scripts/create-region.js

```diff
@@ -0,0 +1,12 @@
+const Region = require('../../models/region.js');
+
+const createRegion = async ()=>{
+    return await Region.bulkCreate([
+        {RegionID: 1, Name:'Brussels'},
```

Comment:
I would add all the 11 provinces: Antwerp, Flemish Brabant, West Flanders, East Flanders, Hainaut, Liège, Limburg, Luxembourg, Namur, Walloon Brabant, Brussels

Reply:
> I would add all the 11 provinces: Antwerp, Flemish Brabant, West Flanders, East Flanders, Hainaut, Liège, Limburg, Luxembourg, Namur, Walloon Brabant, Brussels


okaay :+1:  I'll edit the file and recommit it